spec/restaurant_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'restaurant'
+
+describe Restaurant do 
+  subject(:restaurant) { described_class.new(menu: menu) }
```

Comment:
This is cool.  I've seen this syntax before but not sure what it means. Would be good to get an explanation from you on it.